examples/utils/get-tinylicious-container/src/insecureTinyliciousUrlResolver.ts

```diff
@@ -0,0 +1,68 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IRequest } from "@fluidframework/core-interfaces";
+import {
+    IFluidResolvedUrl,
+    IResolvedUrl,
+    IUrlResolver,
+} from "@fluidframework/driver-definitions";
+import { ITokenClaims } from "@fluidframework/protocol-definitions";
+import jwt from "jsonwebtoken";
+import { v4 as uuid } from "uuid";
+/**
+ * InsecureTinyliciousUrlResolver knows how to get the URLs to the service (in this case Tinylicious) to use
+ * for a given request.  This particular implementation has a goal to avoid imposing requirements on the app's
+ * URL shape, so it expects the request url to have this format (as opposed to a more traditional URL):
+ * documentId/containerRelativePathing
+ */
+export class InsecureTinyliciousUrlResolver implements IUrlResolver {
+    public async resolve(request: IRequest): Promise<IResolvedUrl> {
+        const url = request.url;
+        const documentId = url.split("/")[0];
```

Comment:
nit: You can directly use request.url, don't need the extra variable.

Reply:
sorry i rushed myself...should have been more careful